### DISCUSSION TITLE
Changing the red placeholder background color

### DISCUSSION THREAD

**Discussion by fonzzzzzii** [katoid/angular-grid-layout#113] [Q&A]
How can I change the red background color for the box placeholder as it is being moved? It doesn't seem very straight forward.
![image](https://github.com/katoid/angular-grid-layout/assets/15276772/d6006bb5-ddc3-407a-9eb5-5e0e34de528b)

**Comment by DavidCibin** [comment]
```
// customize placeholder
::ng-deep .ktd-grid-item-placeholder {
    background-color: blue;
}
```

Or you can also use this globally (if you have a custom CSS file that loads last)
```
ktd-grid {
    .ktd-grid-item-placeholder-default {
        background: blue !important;
    }
}
```
<img width="950" alt="Screenshot 2024-05-08 at 10 52 36 AM" src="https://github.com/katoid/angular-grid-layout/assets/78393598/d10cb5a7-fde5-4382-b66b-207d9cfebfe9">
